views/js/test/likertScaleInteractionIMS/data/likert_triple/qti.json

```diff
@@ -0,0 +1,206 @@
+{
+  "identifier": "pci002",
+  "serial": "item_5b07da24f1c5e082350329",
+  "qtiClass": "assessmentItem",
+  "attributes": {
+    "identifier": "pci002",
+    "title": "PCI 002",
+    "label": "",
+    "adaptive": false,
+    "timeDependent": false,
+    "toolName": "TAO",
+    "toolVersion": "3.3.0-sprint77",
+    "class": ""
+  },
+  "body": {
+    "serial": "container_containeritembody_5b07da24f1c50822086021",
+    "body": "\n        \n        <p>This is a sample PCI.<\/p>\n        \n        {{interaction_portablecustominteraction_5b07da250112d182933654}}\n        \n        <p>Here is another one.<\/p>\n        \n        {{interaction_portablecustominteraction_5b07da2501556207762223}}\n        \n        <p>Here is yet another one.<\/p>\n        \n        {{interaction_portablecustominteraction_5b07da2501605590608293}}\n\n    ",
+    "elements": {
+      "interaction_portablecustominteraction_5b07da250112d182933654": {
+        "serial": "interaction_portablecustominteraction_5b07da250112d182933654",
+        "qtiClass": "customInteraction",
+        "attributes": {
+          "responseIdentifier": "likert1"
+        },
+        "debug": {
+          "relatedItem": "item_5b07da24f1c5e082350329"
+        },
+        "choices": {},
+        "typeIdentifier": "likertScaleInteraction",
+        "markup": "\n                    <div class=\"likertScaleInteraction\">\n                        <div class=\"prompt\">\n                            <p>\n                                Do you like movies ? <img src=\"http:\/\/openclipart.org\/image\/300px\/svg_to_png\/35395\/applications-multimedia.png\"><\/img>\n                            <\/p>\n                        <\/div>\n                  <div class=\"scale\">\n      <ul class=\"likert\"><\/ul>\n               <\/div>\n     <\/div>\n                ",
+        "version": "0.0.0",
+        "entryPoint": "likertScaleInteraction\/runtime\/likertScaleInteraction",
+        "libraries": ["likertScaleInteraction\/runtime\/js\/renderer"],
+        "stylesheets": [],
+        "mediaFiles": [],
+        "properties": {
+          "level": "5",
+          "label-min": "Not at all",
+          "label-max": "Very much",
+          "numbers": true
+        },
+        "xmlns": "http:\/\/www.imsglobal.org\/xsd\/portableCustomInteraction"
```

Comment:
**question:** Is it the correct namespace?
**suggestion:**
```suggestion
        "xmlns": "http:\/\/www.imsglobal.org\/xsd\/portableCustomInteraction_v1"
```

Reply:
Ok, updated.